### ISSUE TITLE
Google not redirecting back to my app

### ISSUE THREAD

**Issue by ProgrammerLT** [elsesiy/GAppAuth#4]
Hi,

thanks for this Swift translation GTMAppAuth but I have a problem as Google is not redirecting to my app.

I've set the URL Type corresponding the reverse clientID in Info.plist. It is just that my AppDelegate application(:open:options) is not called.

The authorization endpoint has been reached and an authorisation token has been issued. I was expecting my app to be called with the redirect url, but nothing happens. I'm just back to the view controller where I called GAppAuth.shared.authorize.

I thought maybe I asked for full Gmail scope so my request has to be reviewed, but I scaled back to the default Profile and OpenId scopes and nothing happens.

I wonder if you have an idea about this problem.

Thanks again.

**Comment by elsesiy** [comment]
Hi @ProgrammerLT, have you properly setup the URL schema handling as described in the fourth step in the README? This is non-related to the scopes you pass in, the app should be opened after successful login.

**Comment by ProgrammerLT** [comment]
Hi @elsesiy,

Yes I copy-pasted the reverse URI from Google API Console. I have a previous experience with Outlook using p2/OAuth so I know why it was important.

In the end, it seems to me that Google does not the reverse URI channel to send the Authorization code or the Access token anymore. I went up to look in the AppAuth project and managed to find a Swift implementation of the authorisation process. Finally, I connected to Google and the reverse URI is never used for any kind of communication: authorization, tokens or data.

It's written nowhere in Google so it is kind of strange. 

Thanks for writing back.

**Comment by elsesiy** [comment]
@ProgrammerLT I'm using this library myself and everything works fine so I can't really relate to what you're describing. What has to be changed in order to make it work for you?

**Comment by ProgrammerLT** [comment]
@elsesiy Well, I don't use GAppAuth anymore. I'm using a Swift implementation of AppAuth I found in its GitHub page. 

In fact, I thought Google would be using the redirect URL to communicate the API Access Token but it never did that way. The authorisation code, then the access tokens were exchanged only through HTTP requests. 

I was very confused by your example code because I didn't know if I had to exchange the authorisation code for access token and when I tried it, my HTTP request never came back.
